app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,7 @@
+class FoodsController < ApplicationController 
+
+  def index  
+    @foods = FoodFacade.foods_by_keyword(params[:q])
```

Comment:
This looks good but how could we set up our service to get the count and keyword search with one call? :)